0012-five-bells-ledger-api/0012-five-bells-ledger-api.md

```diff
@@ -0,0 +1,1290 @@
+
+| Resource | Field                            | HTTP Method | API Method       |
+|:---------|:---------------------------------|:------------|:-----------------|
+| Transfer | `ledger` field                   | GET         | [Get Ledger Metadata][] |
+| Transfer | `credits` member `account` field | GET         | [Get Account][]  |
+| Transfer | `debits` member `account` field  | GET         | [Get Account][]  |
+| Message  | `from` field                     | GET         | [Get Account][]  |
+| Message  | `to` field                       | GET         | [Get Account][]  |
+| Message  | `ledger` field                   | GET         | [Get Ledger Metadata][] |
+
+
+### Date-Time
+[Date-Time]: #date-time
+
+All dates and times should be expressed in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) date-time format with precision to the second or millisecond. The time zone offset should always be `Z`, for no offset. In other words, the date-time should be a **string** matching the one of the following formats:
+
+| Precision   | Format                     |
+|:------------|:---------------------------|
+| Second      | `YYYY-MM-DDTHH:mm:ssZ`     |
+| Millisecond | `YYYY-MM-DDTHH:mm:ss.sssZ` |
+
+
+## API Methods
+
+### Get Ledger Metadata
+[Get Ledger Metadata]: #get-ledger-metadata
+
+Receive information about the ledger.
+
+**Authorization:** This method MUST be accessible to all clients authorized as account owners or administrators. This method MAY be accessible to unauthorized users.
+
+#### Request Format
+
+```
+GET /
+```
+
+**[Metadata URL Name][]:** None (top-level path for the ledger)
+
+##### Response Format
+
+A successful result uses the HTTP response code **200 OK** and contains a JSON object with the following fields. (A field name like `foo[].bar` indicates that the row describes the field `bar` in _each_ object contained in the array `foo`.)
+
+| Field               | Value              | Description                       |
+|:--------------------|:-------------------|:----------------------------------|
+| `currency_code`   | String    | Three-letter ([ISO 4217](http://www.xe.com/iso4217.php)) code of the currency this ledger tracks. |
+| `currency_symbol` | String    | Currency symbol to use in user interfaces for the currency represented in this ledger. For example, "$". |
+| `ilp_prefix`        | String             | The ILP Address Prefix of the ledger. This must match the definition from [IL-RFC-15: ILP Addresses](https://github.com/interledger/rfcs/blob/master/0015-ilp-addresses/0015-ilp-addresses.md). |
+| `connectors`        | Array of [URLs][URL] | Accounts belonging to _recommended_ ILP Connectors. Each member of this list MUST be an HTTP(S) URL where a client can [get the connector's ledger account][Get Account]. This list MAY be empty. |
```

Comment:
Actually right now it's an array of objects that each contain `id` (URL) and `name` (username on ledger) fields